main.js

```diff
@@ -61,8 +61,15 @@ class Logger {
 }
 
 const logger = new Logger();
-// set up the logger
-logger.addConsole(process.env.NODE_ENV === 'test' ? 'error' : 'info');
+
+let consoleLoggerLevel;
+if(process.env.NODE_ENV === 'test') {
+	consoleLoggerLevel = 'error';
+} else {
+	consoleLoggerLevel = process.env.CONSOLE_LOGGER_LEVEL || 'info'
```

Comment:
I'd put the default at the lowest level - https://github.com/winstonjs/winston#logging-levels - i.e. `silly`